packages/serverpod_client/lib/src/http/http_status_html.dart

```diff
@@ -1 +1,89 @@
-export 'dart:html' show HttpStatus;
+/**
+ * HTTP status codes.  Extracted from dart:io since it does not exist on package:web/web.dart .
+ */
```

Comment:
Suggestion: Add copyright and license notice.

Please add a copyright and license notice if one is required for this code.